docs/examples/video/video_file_list_outputs.py

```diff
@@ -0,0 +1,71 @@
+#!/bin/env python
```

Comment:
Please convert this to jupyter so we can include this in docs.

Reply:
Also it is not tested (run) now, not mentioned in the docs. In the case of jupyter you would need to only add it to the docs, it should be tested automatically by https://github.com/NVIDIA/DALI/blob/master/qa/TL0_jupyter/test_nofw.sh.